include/ccf/pal/attestation_sev_snp.h

```diff
@@ -198,11 +198,24 @@ pRb21iI1NlNCfOGUPIhVpWECAwEAAQ==
   struct TcbVersionRaw
   {
   private:
-    uint8_t underlying_data[snp_tcb_version_size];
+    uint8_t underlying_data[snp_tcb_version_size]{};
```

Comment:
```suggestion
    uint8_t underlying_data[snp_tcb_version_size]{};
```

What does the trailing `{}` do?